### ISSUE TITLE
Cannot connect to LAN from external

### ISSUE THREAD

**Issue by Reddax** [pivpn/pivpn#141]
Hi,
I've set up PiVPN and when i connect to it externally I can connect to the internet fine but cannot access any of my local machines on the network.

Any ideas?

Client Log:
```
Thu Nov 03 21:47:21 2016 OpenVPN 2.3.13 x86_64-w64-mingw32 [SSL (OpenSSL)] [LZO] [PKCS11] [IPv6] built on Nov  3 2016
Thu Nov 03 21:47:21 2016 Windows version 6.2 (Windows 8 or greater) 64bit
Thu Nov 03 21:47:21 2016 library versions: OpenSSL 1.0.1u  22 Sep 2016, LZO 2.09
Thu Nov 03 21:47:25 2016 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Thu Nov 03 21:47:25 2016 Control Channel Authentication: tls-auth using INLINE static key file
Thu Nov 03 21:47:26 2016 UDPv4 link local: [undef]
Thu Nov 03 21:47:26 2016 UDPv4 link remote: [AF_INET]86.174.166.147:1194
Thu Nov 03 21:47:31 2016 [server] Peer Connection Initiated with [AF_INET]86.174.166.147:1194
Thu Nov 03 21:47:33 2016 do_ifconfig, tt->ipv6=0, tt->did_ifconfig_ipv6_setup=0
Thu Nov 03 21:47:33 2016 open_tun, tt->ipv6=0
Thu Nov 03 21:47:33 2016 TAP-WIN32 device [Ethernet 3] opened: \\.\Global\{FD893E52-CD99-4B24-B54D-B129B2FC27D7}.tap
Thu Nov 03 21:47:33 2016 Notified TAP-Windows driver to set a DHCP IP/netmask of 10.8.0.6/255.255.255.252 on interface {FD893E52-CD99-4B24-B54D-B129B2FC27D7} [DHCP-serv: 10.8.0.5, lease-time: 31536000]
Thu Nov 03 21:47:33 2016 Successful ARP Flush on interface [50] {FD893E52-CD99-4B24-B54D-B129B2FC27D7}
Thu Nov 03 21:47:38 2016 Warning: address 192.168.1.185 is not a network address in relation to netmask 255.255.255.0
Thu Nov 03 21:47:38 2016 ROUTE: route addition failed using CreateIpForwardEntry: The parameter is incorrect.   [status=87 if_index=50]
Thu Nov 03 21:47:38 2016 env_block: add PATH=C:\WINDOWS\System32;C:\WINDOWS;C:\WINDOWS\System32\Wbem
Thu Nov 03 21:47:38 2016 ROUTE: route addition failed using CreateIpForwardEntry: The object already exists.   [status=5010 if_index=50]
Thu Nov 03 21:47:38 2016 env_block: add PATH=C:\WINDOWS\System32;C:\WINDOWS;C:\WINDOWS\System32\Wbem
Thu Nov 03 21:47:38 2016 Initialization Sequence Completed
```

**Comment by HaniKali** [comment]
Hey I just posted the same question earlier today, I finally got it to work, I had to start with a fresh raspbian image again. When running the pivpn installer make sure you highlight the selection and hit spacebar to select an option before proceeding. Also when  it asks if you want to use the same IP select no and then manually put in both the IP and the default gateway, I noticed if you dont do this it doesnt ask for a default gateway. After starting fresh I am now able to connect to internet and am able to SSH to machines on my local lan and access my transmission webpage using local ip addresses. I still am not able to access the local samba shares on the network but i think this might be a windows issue. Eitherway it works for what I need it for both on my phone and laptop. hope this helps.
